charts/renovate/Chart.yaml

```
@@ -24,10 +24,11 @@ maintainers:
 annotations:
   artifacthub.io/license: AGPL-3.0-only
   artifacthub.io/images: |
-    - name: renovate
+    - name: ghcr.io/renovatebot/renovate
```

```suggestion
    - name: ghcr.io/renovatebot/renovate:36.10.1
```
@JamieMagee  should we also add the version to the image name?

---

ok, this needs an renovate config update

https://github.com/renovatebot/helm-charts/blob/06fbbb9ba6852757a8a7b477761069bbd5725787/renovate.json#L28